translations/monero-core_it.ts

```diff
@@ -986,7 +986,7 @@
     <message>
         <location filename="../pages/Settings.qml" line="156"/>
         <source>Rescan wallet balance</source>
-        <translation>Riscannerizza bilancio portafoglio</translation>
+        <translation>Riscannerizza saldo portafoglio</translation>
```

Comment:
"Riscannerizzare" should be "Rescansionare"

Reply:
Yes, you're right. Agreed,